.travis.yml

```diff
@@ -89,6 +90,8 @@ matrix:
       if: 'tag IS NOT present AND type != pull_request'
       os: linux
       script: make integration
+      after_success:
+        - make cov && bash <(curl -s https://codecov.io/bash)
```

Comment:
Will cov runs all unit tests? I think we can use it to replace the default `make test` in `Tests on Linux`.

Reply:
Yes, It will run all the unit tests.

> I think we can use it to replace the default make test in Tests on Linux.

ok